describe/DOType.ts

```diff
@@ -0,0 +1,122 @@
+import { DADescription, describeDA } from "./DADescription.js";
+import { NamingDescription, describeNaming } from "./Naming.js";
+import { SDODescription, describeSDO } from "./SDODescription.js";
```

Comment:
Am I misremembering or did you remove the "describe" at the beginning of function names from the other features? Just want to make sure the style is consistent across files.

Reply:
I do add to describe to functions not exported. All others are of the signature we have agreed on.